web/public/js/services/user-utils.js

```diff
@@ -105,5 +105,22 @@ angular.module('cpZenPlatform').factory('userUtils', ['$location', '$window', '$
     return  avatar || overallDefault;
   }
 
+  userUtils.getTitleForUserTypes = function (userTypes) {
+    var title = [];
+    if (_.includes(userTypes, 'attendee-u13')) {
+      title.push($translate.instant('Ninja (Under 13)'));
```

Comment:
Disagree, it's based on the context.
When joining, if i remember well, we tok the "attendee" keyword because it makes sense, it's the activity you're practicing in the dojo.
I bet in this case, if it's supposed to be generic, Youth sounds fine?